### ISSUE TITLE
Add List of Speakers and Talk Links

### ISSUE THREAD

**Issue by ajpiano** [jquery/2015-developer-summit#25]
- [ ] Anne-Gaelle Colom Impact of Open Source
- [ ] Adam Sontag   Intro "Fixing a simple bug narrative"
- [ ] [Bug Triage Slides](https://docs.google.com/presentation/d/1BJC8If2jp9qOS7gYpJgXD6eeKykD28nJdQH7V7dATjs/edit#slide=id.p) Dave Methvin Reporting and Triaging Bugs (The bug ends up being legit, but core team can't fix right away. "Patches Welcome," they say
- [ ] Stephanie Morillo?    Editing Markdown in Github/Contributing to docs
- [PR Slides ](http://ashleygwilliams.github.io/life-of-a-PR/#1) Ashley Williams   Pull Requests, Code Review, Automated Checks (CLA, CI, etc)| Open Source licenses and CLAs
- [ ] [CLI, GIT & zsh](http://slides.com/brianloveswords/jquery-dev-summit-2015#/) Brian Brennan    Command Line + Git  + Pull Request- How it's used in open source, how it came, the fact that you don't have to be a cli GENIUS to use it for oss | 
- [ ] Adam Ulvi How does this all go live -> VMs/vagrant
- [ ] Nick Hehr Contributing Code
- [ ] Leo Balter    Unit Tests
- [ ] Sarah Frisk   Project Communication, Open Source principles applied to the workplace/your career
- [ ] Rushaine McBean   Community/Inspirational/Aspirational - "Soft" ways to get involved (meetups, conferences, speaking etc)
- [ ] István Szmozsánszky Flaki Browser Dev Tools Workshop

**Comment by kborchers** [comment]
I agree that we need this info collected somewhere. We do already have our [event site](https://events.jquery.org/2015/developer-summit/) as well so where ever this information lives there should be links between these sites directing folks.

**Comment by ajpiano** [comment]
We can port the links and such over to the events.jquery.org repo after - we're working on doing it in this repo for now :)

**Comment by kborchers** [comment]
:+1: Sure, this was really more of a note so we remember to do that :smile_cat:

**Comment by gtewallace** [comment]
any update? I'm pulling the blog together and would love to link to the presentations if possible.  thanks

**Comment by HipsterBrown** [comment]
Link to my slides: https://speakerdeck.com/hipsterbrown/contributing-code-empathetically

**Comment by gtewallace** [comment]
thank you!

**Comment by dmethvin** [comment]
Here are mine:
https://docs.google.com/presentation/d/1BJC8If2jp9qOS7gYpJgXD6eeKykD28nJdQH7V7dATjs/edit#slide=id.p

**Comment by gtewallace** [comment]
thanks Dave - I asked Anne for hers - also need both the Adam's, Leo's, Sarah, Rushaine and Istvan's

**Comment by agcolom** [comment]
My slides can be found at https://www.slideshare.net/secret/ExfDRKk5QRkVwp

**Comment by gtewallace** [comment]
Great! Merci

**Comment by leobalter** [comment]
My slides are here:
- PDF: https://www.dropbox.com/s/b7ges15vl9pgi4g/Unit%20Testing.pdf?dl=0
- Slideshare: http://www.slideshare.net/leobalter/unit-testing-55895845
- The Coding Dojo repository: https://github.com/leobalter/hanoi

**Comment by sfrisk** [comment]
Just noticed this thread (many months later)

Here are my slides: http://www.sarahfrisk.com/slides/open-source.pdf

On Mon, Dec 7, 2015 at 9:46 AM, Leo Balter notifications@github.com wrote:

> My slides are here:
> - PDF: https://www.dropbox.com/s/b7ges15vl9pgi4g/Unit%20Testing.pdf?dl=0
> - Slideshare: http://www.slideshare.net/leobalter/unit-testing-55895845
> - The Coding Dojo repository: https://github.com/leobalter/hanoi
> 
> On Sun, Dec 6, 2015 at 12:14 PM, Greg Wallace notifications@github.com
> wrote:
> 
> > Great! Merci
> > On Dec 6, 2015 11:59 AM, "Anne-Gaelle Colom" notifications@github.com
> > wrote:
> > 
> > > My slides can be found at
> > > https://www.slideshare.net/secret/ExfDRKk5QRkVwp
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> > 
> > https://github.com/jquery/2015-developer-summit/issues/25#issuecomment-162328871
> > 
> > > .
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/jquery/2015-developer-summit/issues/25#issuecomment-162329912
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jquery/2015-developer-summit/issues/25#issuecomment-162544910
> .

## 

Sarah Frisk
sarahfrisk.com

Twitter: twitter.com/SarahFrisk
LinkedIn: linkedin.com/in/sarahfrisk
Github: github.com/sfrisk
